package.json

```diff
@@ -25,6 +25,8 @@
     "@mapbox/mapbox-gl-draw": "^1.1.2",
     "@mapbox/mapbox-gl-draw-static-mode": "^1.0.1",
     "@tinymce/tinymce-vue": "^3.2.8",
+    "@trpc/client": "^11.0.0-next-beta.264",
+    "@trpc/server": "^11.0.0-next-beta.264",
```

Comment:
Does the vue app need to depend on the server?

Reply:
yeah, unfortunately you can't just use the client